model/riscv_insts_base.sail

```diff
@@ -70,11 +70,11 @@ function clause execute (RISCV_JAL(imm, rd)) = {
       ext_handle_control_check_error(e);
       RETIRE_FAIL
     },
-    Ext_ControlAddr_OK(target) => {
+    Ext_ControlAddr_OK(virtaddr(target)) => {
       /* Perform standard alignment check */
       if bit_to_bool(target[1]) & not(extensionEnabled(Ext_Zca))
       then {
-        handle_mem_exception(target, E_Fetch_Addr_Align());
+        handle_mem_exception(virtaddr(target), E_Fetch_Addr_Align());
```

Comment:
Hmm it's unfortunate that we have to destructure to `xlenbits` to access the bits and then reconstruct the type. @Alasdair since a newtype is always a known type is there any way we can access the inner type without destructuring it? E.g. in Rust you have this idiom:

```
struct virtaddr(xlenbits);
```

And then you can access the `xlenbits` via `my_virt_addr.0`.

Oh actually @KotorinMinami  I guess we could use your `virtaddr_bits` function like:

```
    Ext_ControlAddr_OK(target) => {
      /* Perform standard alignment check */
      if bit_to_bool(virtaddr_bits(target)[1]) & not(extensionEnabled(Ext_Zca))
      then {
        handle_mem_exception(target, E_Fetch_Addr_Align());
```

It's a bit verbose though. Hmm.